packages/snaps-simulator/package.json

```diff
@@ -0,0 +1,132 @@
+    "@chakra-ui/react": "^2.6.1",
+    "@emotion/react": "^11.10.8",
+    "@emotion/styled": "^11.10.8",
+    "@ethersproject/units": "^5.7.0",
+    "@metamask/base-controller": "^2.0.0",
+    "@metamask/eth-json-rpc-middleware": "^11.0.0",
+    "@metamask/key-tree": "^7.0.0",
+    "@metamask/permission-controller": "^3.2.0",
+    "@metamask/rpc-errors": "^5.1.1",
+    "@metamask/rpc-methods": "workspace:^",
+    "@metamask/snaps-controllers": "workspace:^",
+    "@metamask/snaps-ui": "workspace:^",
+    "@metamask/snaps-utils": "workspace:^",
+    "@metamask/utils": "^5.0.0",
+    "@minoru/react-dnd-treeview": "^3.4.4",
+    "@reduxjs/toolkit": "^1.9.5",
+    "date-fns": "^2.30.0",
+    "fast-deep-equal": "^3.1.3",
+    "framer-motion": "^10.12.8",
+    "json-rpc-engine": "^6.1.0",
+    "json-rpc-middleware-stream": "^4.2.0",
+    "monaco-editor": "^0.38.0",
+    "pump": "^3.0.0",
+    "react": "^18.2.0",
+    "react-dnd": "^16.0.1",
+    "react-dom": "^18.2.0",
+    "react-hook-form": "^7.43.9",
+    "react-markdown": "^6.0.3",
+    "react-monaco-editor": "^0.52.0",
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.11.1",
+    "redux": "^4.2.1",
+    "redux-saga": "^1.2.3"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.1.0",
+    "@metamask/eslint-config": "^11.0.0",
+    "@metamask/eslint-config-browser": "^11.1.0",
+    "@metamask/eslint-config-jest": "^11.0.0",
+    "@metamask/eslint-config-nodejs": "^11.0.1",
+    "@metamask/eslint-config-typescript": "^11.0.0",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
+    "@redux-saga/is": "^1.1.3",
+    "@redux-saga/symbols": "^1.1.3",
+    "@swc/core": "^1.3.56",
+    "@swc/jest": "^0.2.26",
+    "@testing-library/react": "^14.0.0",
+    "@types/jest": "^27.5.1",
+    "@types/node": "^17.0.36",
+    "@types/pump": "^1.1.1",
+    "@types/react": "^18.2.5",
+    "@types/react-dom": "^18.2.3",
+    "@types/webpack-env": "^1.18.0",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
+    "@typescript-eslint/parser": "^5.42.1",
+    "assert": "^2.0.0",
+    "constants-browserify": "^1.0.0",
+    "css-loader": "^6.7.3",
+    "eslint": "^8.27.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.1.5",
+    "eslint-plugin-jsdoc": "^39.6.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "favicons": "^7.1.2",
+    "favicons-webpack-plugin": "^6.0.0",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^29.0.2",
+    "jest-environment-jsdom": "^29.5.0",
+    "jest-fetch-mock": "^3.0.3",
+    "jest-it-up": "^2.0.0",
+    "lint-staged": "^12.4.1",
+    "monaco-editor-webpack-plugin": "^7.0.1",
+    "prettier": "^2.7.1",
+    "prettier-plugin-packagejson": "^2.2.11",
+    "process": "^0.11.10",
+    "react-refresh": "^0.14.0",
+    "readable-stream": "^4.4.0",
```

Comment:
Realizing now that we are bringing in a newer version of readable-stream than Browserify. Do we want to match its versions for the things we polyfill? 

It will also potentially simplify our LM policy diffs.

Reply:
Not sure what you mean. Which version should it be instead?

Reply:
Seems that we use `3.6.2` everywhere else?